solution/ui/regulations/js/src/components/SupplementalContent.vue

```diff
@@ -132,6 +135,20 @@ export default {
             }
             return `Subpart ${this.subparts[0]}`;
         },
+
+        resourceLink: function () {
+            let qString = "/resources\\?title=42&part=" + this.part
+
+            if (this.activePart.includes("Subpart")){
+                qString = qString + "&subpart="+this.part + "-" + this.params_array[1][1] 
```

Comment:
I know it's just a personal preference but I like to use template literal strings for situations like this instead of string concatenation.

Reply:
changed

Reply:
nice